client/css/style.css

```diff
@@ -1108,18 +1108,23 @@ background on hover (unless active) */
 }
 
 .scroll-down-arrow {
-	width: 48px;
-	height: 48px;
-	line-height: 48px;
+	width: 36px;
+	height: 36px;
+	line-height: 34px;
 	border-radius: 50%;
-	background: #fff;
-	color: #333;
-	border: 1px solid #84ce88;
+	background: var(--window-bg-color);
+	color: #84ce88;
+	border: 2px solid #84ce88;
 	text-align: center;
+	transition: background 0.2s, color 0.2s;
+	box-shadow:
+		0 3px 5px -1px rgba(0, 0, 0, 0.2),
+		0 6px 10px 0 rgba(0, 0, 0, 0.14);
```

Comment:
Stole this from Material-UI for floating buttons. They have 3 box-shadows, I removed the last one which made the button shadow too thick and present compared to the rest of our UI, but re-used the other 2 shadows as-is. I tried to remove one then the other, it doesn't look as nice imo.